packages/app/src/cli/prompts/config.test.ts

```diff
@@ -200,7 +200,10 @@ describe('confirmPushChanges', () => {
 
       configuration.name = 'app2'
       configuration.access_scopes = {scopes: 'read_themes, read_customers'}
-      configuration.webhooks.api_version = 'unstable'
+      configuration.webhooks = {
+        ...configuration.webhooks,
+        api_version: 'unstable',
+      }
```

Comment:
`DEFAULT_CONFIG` includes `webhoooks` information so I have just modified the required field

Reply:
but `DEFAULT_CONFIG` only has `api_version` and since you are setting `api_version: 'unstable'` just below, having that line doesn't do anything no?

Or we can keep it for future usage if we are going to add more fields